libs/langgraph/src/pregel/types.ts

```diff
@@ -24,6 +24,11 @@ export type PregelInputType = any;
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type PregelOutputType = any;
 
+export type Promisified<T extends (...args: unknown[]) => unknown> =
```

Comment:
I think the goal of this file is to export types we want to expose in the public api of the library. utility types should either be defined in the file they're used or in some private file